### PR TITLE
JIRA: ABC-132 Adjusting start value of time range in AlembicStreamPlayer slighly changes the end value

### PR DESCRIPTION
**Pull request by clusty**
Bug caused by slight inaccuracies of the MinMaxSlider widget.

### CODE REVIEW

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2021.2.0a17 (Windows only)

**Review by judubu — APPROVED** [review]
👍

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that caused unintended end of range stream edits.
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/CHANGELOG.md:11]: Suggestion:
`Fixed an issue where changing the start value of the streaming time range would slightly change the end of range value.`

**Review by sebastienduverne — APPROVED** [review]
Changelog ok.

**Pull request merged**